AlphaWallet/KeyManagement/EtherKeystore.swift

```diff
@@ -142,15 +143,23 @@ open class EtherKeystore: Keystore {
         }
     }
 
-    //TODO improve
-    static var current: Wallet? {
-        do {
-            return try EtherKeystore(analyticsCoordinator: nil).recentlyUsedWallet
-        } catch {
-            return .none
+    var currentWallet: Wallet {
+        //Better crash now instead of populating callers with optionals
+        if let wallet = recentlyUsedWallet {
+            return wallet
+        } else if wallets.count == 1 {
+            return wallets.first!
```

Comment:
Always knew you loved force unwrap ;)

Reply:
Haha. Jokes aside, that’s the least amount of change to make the PR work :)